include/zephyr/arch/common/sys_bitops.h

```diff
@@ -62,19 +62,19 @@ static ALWAYS_INLINE
 	/* Doing memory offsets in terms of 32-bit values to prevent
 	 * alignment issues
 	 */
-	sys_set_bit(addr + ((bit >> 5) << 2), bit & 0x1F);
+	sys_set_bit(addr + ((bit >> 5) << 2U), bit & 0x1F);
```

Comment:
Is it needed to have 2U here? Is this implementation defined?

Reply:
There is no need to use `2U` here, I just added it for consistency. Same question was asked in the upstream PR: https://github.com/zephyrproject-rtos/zephyr/pull/86624